src/main/java/com/emc/mongoose/storage/driver/pravega/kvs/PravegaKVSDriver.java

```diff
@@ -409,7 +426,7 @@ private boolean submitCreate(O op) {
             KVTFactoryCreateFunctionImpl::new);
         // create the kvt factory if necessary
         val kvtFactory = kvtFactoryCache.computeIfAbsent(scopeName, kvtFactoryCreateFunc);
-        val kvtClientCreateFunc = kvtClientCreateFuncCache.computeIfAbsent(kvtFactory, KVTClientCreateFunctionImpl::new);
+        val kvtClientCreateFunc = kvtClientCreateFuncCache.computeIfAbsent(kvtFactory, KVTForCreateClientCreateFunctionImpl::new);
```

Comment:
why do we need another word create in the variable name? It's a KVT create function, but it's a kvt for client, so it's KVTClient create function. If it sounds better to you, we can call it KVTCreateClientFunctionImpl. But that to me implies that it's some sort of the client function and not the client kvt.

Reply:
Oh, I got it. ForCreate and ForRead. Can we add Op to them then? KVTForCreateOpClientCreateFunctionImpl? Though at this point it does look like passing a serializer might be a prettier choice. I'm personally ok though with this, except for the name